docs/source/optimization_ov.mdx

```diff
@@ -0,0 +1,115 @@
+quantization_config = OVConfig()
+# Instantiate our OVQuantizer using the desired configuration
+quantizer = OVQuantizer.from_pretrained(model)
+# Create the calibration dataset used to perform static quantization
+calibration_dataset = quantizer.get_calibration_dataset(
+    "glue",
+    dataset_config_name="sst2",
+    preprocess_function=partial(preprocess_function, tokenizer=tokenizer),
+    num_samples=300,
+    dataset_split="train",
+)
+# Apply static quantization and export the resulting quantized model to OpenVINO IR format
+quantizer.quantize(
+    quantization_config=quantization_config,
+    calibration_dataset=calibration_dataset,
+    save_directory=save_dir,
+)
+# Save the tokenizer
+tokenizer.save_pretrained(save_dir)
+```
+
+After applying static quantization on our model, we can then easily load it with our [OVModel](https://huggingface.co/docs/optimum/intel_optimization_ov) classes and perform inference with OpenVINO Runtime.
+
+```python
+from transformers import pipeline
+from optimum.intel.openvino import OVModelForSequenceClassification
+
+ov_model = OVModelForSequenceClassification.from_pretrained(save_dir)
+cls_pipe = pipeline("text-classification", model=ov_model, tokenizer=tokenizer)
+text = "He's a dreadful magician."
+outputs = cls_pipe(text)
+[{'label': 'NEGATIVE', 'score': 0.9840195178985596}]
+```
+
+Quantization aware training (QAT) can also be applied in order to simulate the effects of quantization during training, in order to alleviate its effects on the model's accuracy. In the case where post-training quantization results in an import accuracy degradation, QAT can be used instead.
+Here is an example on how to fine-tune a DistilBERT on the sst-2 task while applying quantization aware training (QAT).
+
+```python
+import numpy as np
+from datasets import load_dataset, load_metric
+from transformers import AutoModelForSequenceClassification, AutoTokenizer, TrainingArguments, default_data_collator
+
+model_id = "distilbert-base-uncased-finetuned-sst-2-english"
+model = AutoModelForSequenceClassification.from_pretrained(model_id)
+tokenizer = AutoTokenizer.from_pretrained(model_id)
+dataset = load_dataset("glue", "sst2")
+dataset = dataset.map(
+    lambda examples: tokenizer(examples["sentence"], padding=True, truncation=True, max_length=128), batched=True
+)
+metric = load_metric("accuracy")
+compute_metrics = lambda p: metric.compute(predictions=np.argmax(p.predictions, axis=1), references=p.label_ids)
+```
+```diff
+-from transformers import Trainer
++from optimum.intel.openvino.trainer import OVConfig, OVTrainer
+
+# Load the default quantization configuration detailing the quantization we wish to apply
++ov_config = OVConfig()
+
+-trainer = Trainer(
++trainer = OVTrainer(
+    model=model,
+    args=training_args,
+    train_dataset=train_dataset,
+    eval_dataset=eval_dataset,
+    compute_metrics=compute_metrics,
+    tokenizer=tokenizer,
+    data_collator=data_collator,
```

Comment:
training_args, train_dataset, eval_dataset and data_collator are not defined here. It would be nice if this can be modified so it is an example that you can copy/paste and then just works.

Reply:
It's a good point, I have modified the example accordingly